src/main/java/com/fasterxml/jackson/core/Base64Variant.java

```diff
@@ -154,6 +165,11 @@ public Base64Variant(Base64Variant base, String name, int maxLineLength)
      * line length) differ
      */
     public Base64Variant(Base64Variant base, String name, boolean usesPadding, char paddingChar, int maxLineLength)
+    {
```

Comment:
Please add `@since 2.12` for new non-private methods (helps with Javadocs wrt public methods; internal changes for other types).

Reply:
This particular highlighted constructor is not new, So I haven't added the @since prefix for this.